20171102/AGENDA.md

```diff
@@ -0,0 +1,9 @@
+# Sardana follow-up meeting
+To be held on 02.11.2017 at 14:00
+
+# Agenda
+1. Introduce <feature/limitation> and share experience about it.
+2. Report user's experience - round table.
+3. Propose new development, enhancement or bug fix, for collaborative
+   development
```

Comment:
Do you think that we should already propose these developments beforehand or we just discuss them during the meeting? I think that one per each institue is a good starting point. We could also choose one most priority house keeping task. I would start with classifying issues with the [labels](https://help.github.com/articles/about-labels/).

Reply:
Hi Zibi,
               I am not sure I understand your question. For me it is fine if you keep your points in the agenda, without any concrete development.

Reply:
I propose that after the meeting we should have one, the most priority, proposal of new development (enhancement of bug fix) per each insitutue - four in total. So after the meeting we  could collaborate on them between the institutes until the next followup meeting with the aim of submitting a PR or even having it integrated in the develop branch. So the question is:
* should we add these to the agenda before the meeting?
* or we just decide them during the meeting and later on put them in the minutes?

Knowing them beforehand we could refresh our knowldge about a given topic and preper ourseves for the discussion...

Apart from these 4 developments, I propose to select one "house keeping" task that we could share between all of us as well. This could be some task related to the continuous integration, github project maintenance, etc. Something that maybe is not so high priority for our users, but would faciliate our work. I popose to start using labels and classify already existing issues.

Reply:
I think we could decide them during the meeting, each of us could think about one or
 more and discussing we could figure out which one could have more priority or could
 be more interesting to develop in collaboration.